### ISSUE TITLE
Crop ages block loots

### ISSUE THREAD

**Issue by Replaycchi** [PluginBugs/Issues-ItemsAdder#1434]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Describe the solution you'd like

Hello, I want to suggest you to add the fact of dropping some resources when you break wheat or other crops only when it is fully grown.
Would be something like this :
```
loots:
  blocks:
    ble:
      type: WHEAT-7
      items:
        bledor:
          item: XXXX:XXXX
          min_amount: 1
          max_amount: 1
          chance: 100
```
Thanks 

### Is your feature request related to a problem?

Without it, players could break the wheat over and over again to get the drops without it even growing

### Describe alternatives you've considered

```
loots:
  blocks:
    ble:
      type: WHEAT-7
      items:
        bledor:
          item: cinderionfermier:bledor
          min_amount: 1
          max_amount: 1
          chance: 100
```

or

```
loots:
  blocks:
    ble:
      type: WHEAT
      age: 7
      items:
        bledor:
          item: cinderionfermier:bledor
          min_amount: 1
          max_amount: 1
          chance: 100
```

**Comment by LoneDev6** [comment]
Is this a bug report?
The plugin already checks the age of the crop and only drops loots if the age equals to its max value.

**Comment by Replaycchi** [comment]
It isn't a bug report, i want to edit the loot table of vanilla wheat to add my IA's item like this example but for Wheat, carrots potatoes and beetroots.

`loots:
  blocks:
    nether_quartz_ore:
      type: NETHER_QUARTZ_ORE
      items:
        crystal:
          item: itemsadder:crystal
          min_amount: 1
          max_amount: 2
          chance: 10
        knowledge_fragment:
          item: itemsadder:knowledge_fragment
          min_amount: 1
          max_amount: 2
          chance: 15`

**Comment by LoneDev6** [comment]
Did you try to create a loot like that?
They should drop only if the age is at its max value.
```yml
loots:
  blocks:
    ble:
      type: WHEAT
      items:
        bledor:
          item: cinderionfermier:bledor
          min_amount: 1
          max_amount: 1
          chance: 100
```

**Comment by Replaycchi** [comment]
I've tried but it drop regardless of age, so i guess it's a bug.

**Comment by LoneDev6** [comment]
I'll investigate, thanks

**Comment by LoneDev6** [comment]
Should be fixed in the upcoming update 3.0.0
